proposals/forms.py

```diff
@@ -630,3 +630,27 @@ def clean(self):
                     self.add_error('comments', _(
                         'Informatiebrief voor traject {} nog niet toegevoegd.').format(
                         study.order))
+    
+
+class TranslatedConsentForms(forms.ModelForm):
+
+    class Meta:
+        model = Proposal
+        fields = ['translated_forms', 'translated_forms_languages']
+        widgets = {
+            'translated_forms': forms.RadioSelect(choices=YES_NO),
+        }
+    
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+    
+    def clean(self):
+        cleaned_data = super(TranslatedConsentForms, self).clean()
+
+        if cleaned_data['translated_forms'] is None:
+            self.add_error('translated_forms', _('Dit veld is verplicht om '
+                                                'verder te gaan.'))
+            
+        elif cleaned_data['translated_forms'] == True and not cleaned_data['translated_forms_languages']:
+            self.add_error('translated_forms_languages', _('Vul in in welke talen de formulieren '
```

Comment:
These lines are also too long, and not my preference in indentation. You're jumping from an indent of 12 to 48 lines! I would much prefer

    self.add_error(
        'translated_forms_languages',
        _('Vul in in welke talen de formulieren worden vertaald.'),
    )

This also saves you from breaking up the string.